liferay-echo/liferay-portal
# LPS-187909 add item selector and change conditions for displaying the selectors

Motivation
=======
Show the "More Experiences" button or "More segments"if there are more than 8 items in each selector. That "More Experiences" button will open item selector.

We have also changed the conditions for displaying the segments and experiences selectors

[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-187909)


Steps to Verify:
----------------
1. Set feature.flag.LPS-186155=true in portal-ext.properties.
1. Open the simulation panel and check that only the first selector is available if we don't have segments or experiences.
2. Create some segments in People -> Segments and come back to the simulation panel. Check that the segment selector is now available.
3. Do the same with experiences.
4. Check that if we have more than 8 segments the "More segments" button is present and it opens the segments item selector.
5. Check that if we have more than 8 experiences the "More experiences" button is present and it opens the experiences item selector.
6. Check that everything keeps working fine without the feature flag.
